test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/DiscoverTests.cs

```diff
@@ -263,10 +260,12 @@ public async Task CancelTestDiscovery(RunnerInfo runnerInfo)
         await Task.Run(() => _vstestConsoleWrapper.DiscoverTests(testAssemblies, runSettingsXml, discoveryEvents.Object));
 
         // Assert.
-        Assert.IsTrue(isTestCancelled, "Discovery was not cancelled");
-        var done = sw.Elapsed;
-        var timeTillCancelled = done - cancellationCalled;
-        timeTillCancelled.Should().BeLessThan(2.Seconds());
+        Assert.IsTrue(isTestCancelled);
```

Comment:
Keep this change, it is useful.

```suggestion
        Assert.IsTrue(isTestCancelled, "Discovery was not cancelled");
```